spec/unit/network/http/connection_spec.rb

```diff
@@ -250,23 +250,23 @@ def retry_after(datetime)
   describe 'connection request errors' do
     it "logs and raises generic http errors" do
       generic_error = Net::HTTPError.new('generic error', double("response"))
-      expect_any_instance_of(Net::HTTP).to receive(:request).and_raise(generic_error)
+      stub_request(:get, url).and_raise(generic_error)
```

Comment:
On the master branch, we have `to_raise` rather than `and_raise`

https://github.com/puppetlabs/puppet/blob/master/spec/unit/network/http/connection_spec.rb#L253

Reply:
ah yes, good catch, thanks!

Reply:
ah it looks like those [methods are aliased](https://github.com/bblimke/webmock/blob/d8686502442d9830dcccd24a1120ac08413d857a/lib/webmock/request_stub.rb#L37), but agree it's better to stick with one